custom/icds_reports/migrations/sql_templates/update_tables50.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE agg_awc ADD COLUMN aww_present_vhnd integer;
+ALTER TABLE agg_awc ADD COLUMN vit_a_given integer;
+ALTER TABLE agg_awc ADD COLUMN anc_check_conducted integer;
```

Comment:
These tables are not managed by django, so we can't use django's migration framework to modify them. That is mostly a historical reality, they were not created in django so they are not there now. I am not sure if there was a good reason for it at the time, although they predate the dashboard being in django (they are from the tableau days) so it is possible this was easier at the time. I tried getting them to be managed by django when I created the django models for them (those used to not exist and all data access was done with sqlagg instead of the django ORM until a year or so ago), but django doesnt handle compound primary keys well, so I contented myself with the idea that creating the models was a good enough start. I think its still a good change to make, and almost certainly possible even with our weird primary keys, but it is pretty low priority.

Reply:
Roger. That all makes sense and thanks for the detailed explanation.